example/lambdas/python-processing/package.json

```diff
@@ -10,10 +10,12 @@
   },
   "scripts": {
     "test": "python -m unittest",
```

Comment:
Doesn't this need to run in the pipenv?

Reply:
fair point

Reply:
https://github.com/nasa/cumulus/pull/2038/commits/af84837bf9810ba12c4f0a85ed99d8b0d39d0258 👍